processing_TEMPLATE.yaml

```diff
@@ -38,4 +40,12 @@ tagging:
     min_frames: 3
 
     # If tracks moves this many pixels in any direction then we shall assume it isn't a false positive
-    animal_movement: 50
\ No newline at end of file
+    animal_movement: 50
+
+
+audio:
+    convert_workers: 2
+    analysis_workers: 2
+
+    # The command will be called to perform analysis on audio recordings (e.g. Cacophony Index, speech detection)
+    analysis_command: 'docker run -v "{folder}":/io cacophonyproject/audio-analysis /io/"{basename}"'
```

Comment:
maybe put the thermal stuff inside it's own nested object as well
```
thermal:
  classify:....
```

Reply:
Yep, I want to do this but want to do that in a separate PR.

Reply:
https://github.com/TheCacophonyProject/cacophony-processing/issues/54